resources/translations/en.edn

```diff
@@ -143,7 +143,7 @@
          :apply-for-bundle "Apply for these %1 resources as a bundle "
          :empty [:div.intro [:p "Select resources to apply for."]]
          :header "%1 resources in cart"
-         :intro [:div.intro [:p "Here are all the resources that you have picked. If the resources can be applied with the same application, they will be automatically grouped into a bundle."]]
+         :intro [:div.intro [:p "Here are the resources that you have selected. If the resources can be applied to with the same application, they will be automatically grouped into a bundle."]]
```

Comment:
My LB simulator proposes the following:

"Here are the resources that you have selected." -> "Here are the selected resources."

"If the resources can be applied to with the same application [...]" -> "If several resources can be applied to using the same application [...]"

These are just suggestions that I think are slight improvements: if you like your version more, I'm fine with that.

Reply:
Good suggestions!